common/src/Admin.php

```diff
@@ -88,6 +88,10 @@ public function options_page() {
             }
 
             if (!$errors) {
+                $nonce = $_REQUEST['_wpnonce'];
+                if ( ! wp_verify_nonce( $nonce, 'submit_trustmark_settings' ) ) {
+                    exit;
+                }
```

Comment:
We're not following WordPress code standards, so please remove the extra spaces here.

`$_REQUEST['_wpnonce']` may not be set or it may not be a string. So write your condition like this:

~~~php
if (empty($_REQUEST['_wpnonce']) || !wp_verify_nonce((string) $_REQUEST['_wpnonce'], ...))
~~~

Send a proper HTTP status code using `http_response_code`. I suggest 400 (Bad Request).

Make clear what happened:

~~~php
die("Invalid nonce");
~~~